LondonBritishLibrary/orient/BLorient818.xml

```diff
@@ -1,15 +1,19 @@
-<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
 schematypens="http://relaxng.org/ns/structure/1.0"
-?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
    schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:lang="en" xml:id="BLorient818" type="mss">
     <!-- This is an expanded miniMs record: many fields are still unfilled (MV 2018-02-15) -->
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title>Kǝbra nagaśt, Chronicle of John of Nikiu, History of the Blessed at the time of Jeremiah, Zenāhu la-ʾƎskǝndǝr, Zenā Sǝkǝndǝs, Maṣḥafa ṭǝnta hāymānot, 
-                    Fǝkkāre ʾIyasus, Apocalypse of Mary, Ṭǝbāba Sābelā, Physiologus, Raʾǝya Sinodā, Letter of Abgar, Homilies
+                <title>Kǝbra nagaśt, Yoḥnnǝs Madabbǝr, Zenā bǝḍuʿān ba-mawāʿǝla ʾErmǝyās, Zenāhu la-ʾƎskǝndǝr, Zenā Sǝkǝndǝs, Maṣḥafa ṭǝnta hāymānot,
+                    Fǝkkāre ʾIyasus, Rāʾǝya Māryām, Ṭǝbāba Sābelā, Dǝrsān za-bǝṣuʿ Fisālgos,
```

Comment:
Maybe this follows the manuscript, but it would maybe be better to normalise here to bəḍuʿ instead of having versions with ḍ and ṣ in the title